### ISSUE TITLE
Overclock options for more than one coin

### ISSUE THREAD

**Issue by rodrigobragamg** [TrailingStop/TT-Miner-release#27]
Hi.
This is my command line: /TT-Miner -b 127.0.0.1:25076 -u 0xE82CfB6CB0d6b47fBE1f25A322767a380c709Bca.NVIDIA_Rig -o 0xpool.io:80 -a sha3solidity -p fb1988RBA -oc-coreoff 255 -oc-core 1605,1695,1555,1555,1635,1575 -oc-mem 5001 -oc-fan 90,85,87,8785,85 -cZ ZIL -PZ <zil15gnc34xk3v9vpk0e735fq0c7qzglrlue99pjwk>.<NVIDIA_Rig>:<fb1988RBA>@<stratum+tcp://br.crazypool.org>:<5005> -oc-coreoff 0 -oc-coreZIL 1085 -oc-memZIL 11300,11600,10800,10900,11500,11100 -oc-fanZIL 92,90,95,95,90,91
----
The miner error says that the selector for coreoffZIL or coreZIL are not available...
Doesn't your miner let configure different OC's for more than 1 coin?
Or is it just for memory?

Thanks.

**Comment by TrailingStop** [comment]
Hi - just replace the ',' with an space: ' '
-oc-memZIL 11300 11600 10800 10900 11500 11100

**Comment by rodrigobragamg** [comment]
> Hi - just replace the ',' with an space: ' ' -oc-memZIL 11300 11600 10800 10900 11500 11100

Thx for the fast reply.
True, my bad.
But still, the problem persists, the miner returns:
Selector "ZIL" not found, Command line option: "oc-coreZIL"

The new command line:
-oc-coreoff 255
-oc-core 1605 1695 1555 1555 1635 1575
-oc-mem 5001
-oc-fan 90 85 87 87 85 85
-cZ ZIL -PZ <zil15gnc34xk3v9vpk0e735fq0c7qzglrlue99pjwk>.<%WORKER_NAME%>:<fb1988RBA>@<stratum+tcp://br.crazypool.org>:<5005> -oc-coreoff 0 -oc-coreZIL 1085 -oc-memZIL 11300 11600 10800 10900 11500 11100 -oc-fanZIL 92 90 95 95 90 91

**Comment by TrailingStop** [comment]
Yes, you need to keep the same patter in the command line. You choose Z for zil: -cZ ZIL -PZ walllet....etc
so you need to use Z for OC as well - not ZIL!

**Comment by rodrigobragamg** [comment]
Ohh.. I see. I tried to follow the miner help (where you wrote "memRXD") and thought that we could write the coin name freely in the OC patterns. Changed to Z in the end of each OC and worked fine.
Thanks so much. Nice job with the miner... I will switch 3 rigs for it.

**Comment by TrailingStop** [comment]
Thanks for the flowers :)
Yes - my documentation lacks a lot.
